addons/packages/vsphere-cpi/1.20.0/package.yaml

```diff
@@ -5,21 +5,173 @@ metadata:
 spec:
   refName: vsphere-cpi.community.tanzu.vmware.com
   version: 1.20.0
-  releaseNotes: "vsphere-cpi 1.20.0 https://github.com/kubernetes/cloud-provider-vsphere"
+  releaseNotes: vsphere-cpi 1.20.0 https://github.com/kubernetes/cloud-provider-vsphere
   licenses:
-    - "Apache 2.0"
+  - Apache 2.0
   template:
     spec:
       fetch:
-        - imgpkgBundle:
-            image: projects.registry.vmware.com/tce/vsphere-cpi@sha256:7c9d83092f79589bbfb401203668e0f80a9f6c08a84dd70422d5689593cae960
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/vsphere-cpi@sha256:a44e3134ab328c7eeb78019475915f22984746c7dd2e3096047cb3b1e214b337
       template:
-        - ytt:
-            paths:
-              - config/
-        - kbld:
-            paths:
-              - "-"
-              - .imgpkg/images.yml
+      - ytt:
+          paths:
+          - config/
+      - kbld:
+          paths:
+          - '-'
+          - .imgpkg/images.yml
       deploy:
-        - kapp: {}
+      - kapp: {}
+  valuesSchema:
+    openAPIv3:
+      type: object
+      additionalProperties: false
+      description: OpenAPIv3 Schema for vsphere-cpi
+      properties:
+        vsphereCPI:
+          type: object
+          additionalProperties: false
+          properties:
+            tlsThumbprint:
+              type: string
+              default: ""
+              description: The cryptographic thumbprint of the vSphere endpoint's certificate
+            server:
```

Comment:
These 4 items are required. Should we add `required`  to them?

Reply:
Create an issue in YTT repo to illustrate the issue we are facing with nullable annotation
https://github.com/vmware-tanzu/carvel-ytt/issues/581

Reply:
Changed those required fields to default to empty string and removed nullable annotation